caffe2/python/onnx/backend.py

```diff
@@ -960,6 +960,11 @@ def _create_reciprocal(cls, init_model, pred_model, n, opset_version):
             exponent=-1.0,
         )
 
+    @classmethod
+    def _create_matmul(cls, init_model, pred_model, n, opset_version):
+        n.attrs['broadcast'] = 1
+        return cls._common_onnx_node_to_caffe2_op(init_model, pred_model, n, opset_version)
```

Comment:
We have `CommonOnnxNodeToCaffe2Ops` in C++ implementation, too. It might save you some code. 

https://github.com/caffe2/caffe2/blob/master/caffe2/onnx/backend.cc#L437